src/main/groovy/org/shipkit/internal/config/GitOriginRepoProvider.java

```diff
@@ -0,0 +1,32 @@
+package org.shipkit.internal.config;
+
+import org.gradle.api.logging.Logger;
+import org.gradle.api.logging.Logging;
+import org.shipkit.internal.exec.ProcessRunner;
+
+public class GitOriginRepoProvider {
```

Comment:
+1 to new class - makes the code cleaner. Do you think it belongs somewhere in git / vcs packages?